build.sh

```diff
@@ -24,7 +24,7 @@ GO_BUILD_ENV=("CGO_ENABLED=0" "GO_BUILD_FLAGS=${GO_BUILD_FLAGS:-}" "GOOS=${GOOS}
 toggle_failpoints() {
   mode="$1"
   if command -v gofail >/dev/null 2>&1; then
-    run gofail "$mode" server/etcdserver/ server/mvcc/backend/ server/wal/
+    run gofail "$mode" server/etcdserver/ server/mvcc/ server/wal/ server/mvcc/backend/
```

Comment:
`make gofail-enable` and and `toggle_failpoints` in build.sh are separate clashing mechanisms. `toggle_failpoints` will always override `make gofail-enable` as it is run last. You need to either use one or the other.

Reply:
removed `gofail-enable`